src/EditorFeatures/Core.Wpf/QuickInfo/LazyToolTip.cs

```diff
@@ -18,17 +19,17 @@ internal partial class ContentControlService
         /// <summary>
         /// Class which allows us to provide a delay-created tooltip for our reference entries.
         /// </summary>
-        private class LazyToolTip : ForegroundThreadAffinitizedObject
+        private class LazyToolTip : ForegroundThreadAffinitizedObject, IDisposable
```

Comment:
perhaps LazyDisposableToolTip?

Reply:
➡️ Keeping the current name because 1) it's impossible to create an instance of this without realizing it's disposable (the `Create` method returns RCD) and 2) there is no corresponding non-disposable type.